### PR TITLE
Removed, and documented removal of, texture swizzles per decision at rec...

### PR DESCRIPTION
**Pull request by kenrussell**
...ent F2F.

### CODE REVIEW

File: specs/latest/2.0/index.html [hunk 1]
```diff
@@ -650,10 +650,6 @@ <h2 class="no-toc">Table of contents</h2>
   const GLenum ANY_SAMPLES_PASSED_CONSERVATIVE               = 0x8D6A;
   const GLenum SAMPLER_BINDING                               = 0x8919;
   const GLenum RGB10_A2UI                                    = 0x906F;
-  const GLenum TEXTURE_SWIZZLE_R                             = 0x8E42;
-  const GLenum TEXTURE_SWIZZLE_G                             = 0x8E43;
-  const GLenum TEXTURE_SWIZZLE_B                             = 0x8E44;
-  const GLenum TEXTURE_SWIZZLE_A                             = 0x8E45;
   const GLenum GREEN                                         = 0x1904;
   const GLenum BLUE                                          = 0x1905;
```
Comment (Oletus) [inline specs/latest/2.0/index.html]: GREEN and BLUE enums are only used for texture swizzle and so they could also be removed from here.

File: specs/latest/2.0/index.html [hunk 2]
```diff
@@ -2198,6 +2190,10 @@ <h2 class="no-toc">Table of contents</h2>
 
     <h3>New Features Supported in the WebGL 2 API</h3>
 
+    <div class="note editor">
+        Needs many updates for OpenGL ES 3.0 features
+    </div>
```
Comment (Oletus) [inline specs/latest/2.0/index.html]: I don't think that this note is needed - this is under "4 Other differences between WebGL 2.0 and WebGL 1.0", so this section does not need to cover anything that's covered by the above API documentation, and section 4 already has a similar comment at the top.

**Comment by Oletus** [comment]
A couple of small comments, but other than that looks good!

**Comment by jdashg** [comment]
LGTM

File: specs/latest/2.0/index.html (hunk 1 above)
Comment (kenrussell) [inline specs/latest/2.0/index.html]: Thanks very much; will fix.

File: specs/latest/2.0/index.html (hunk 2 above)
Comment (kenrussell) [inline specs/latest/2.0/index.html]: Thanks, I'd missed the nesting of this section. I'll remove this note.

**Comment by kenrussell** [comment]
Thanks for the reviews Olli and Jeff. Addressed review feedback. Merging now.

**Pull request merged**